test/Conversion/ExportVerilog/verilog-basic.mlir

```diff
@@ -595,19 +595,25 @@ hw.module @BindEmission2() -> () {
 }
 
 
-hw.module @bind_rename_port(%.io_req_ready.output: i1, %reset: i1, %clock: i1) {
+hw.module @bind_rename_port(%.io_req_ready.output: i1, %reset: i1 { hw.verilogName = "resetSignalName" }, %clock: i1) {
   // CHECK-LABEL: module bind_rename_port
   // CHECK-NEXT: input _io_req_ready_output,
-  // CHECK-NEXT:       reset,
+  // CHECK-NEXT:       resetSignalName,
   // CHECK-NEXT:       clock
   hw.output
 }
 
+// CHECK-LABEL: module SiFive_MulDiv
 hw.module @SiFive_MulDiv(%clock: i1, %reset: i1) -> (io_req_ready: i1) {
   %false = hw.constant false
   hw.instance "InvisibleBind_assert" sym @__ETC_SiFive_MulDiv_assert @bind_rename_port(".io_req_ready.output": %false: i1, reset: %reset: i1, clock: %clock: i1) -> () {doNotPrint = true}
   hw.probe @__ETC_SiFive_MulDiv_assert, %false, %reset, %clock: i1,i1,i1
   hw.output %false : i1
+  //      CHECK: bind_rename_port InvisibleBind_assert (
+  // CHECK-NEXT:   ._io_req_ready_output (_InvisibleBind_assert__io_req_ready_output),
+  // CHECK-NEXT:   .resetSignalName      (reset),
```

Comment:
Before this change/fix, this would be emitted as:

```
    bind_rename_port InvisibleBind_assert (
      ._io_req_ready_output (_InvisibleBind_assert__io_req_ready_output),
      .resetSignalName                (reset),
      .clock                (clock)
    );
```

Due to measuring `reset` not the emitted `resetSignalName`.  When encountered this, the port actually had an empty name w/`hw.verilogName` setting it to `_GEN` (`_GEN_1`, etc.), so the entire name wasn't included in the measurement.